cmd/osmosisd/cmd/config.go

```diff
@@ -141,6 +144,11 @@ output = "{{ .Output }}"
 node = "{{ .Node }}"
 # Transaction broadcasting mode (sync|async)
 broadcast-mode = "{{ .BroadcastMode }}"
+# Human-readable denoms
+# If enabled, when using CLI, user can input base denoms (baseatom, basescrt, baseweth, basewbtc, basewbtc.grv etc.) instead of their ibc equivalents.
```

Comment:
```suggestion
# Human-readable denoms
# If enabled, when using CLI, user can input base denoms (baseatom, basescrt, baseweth, basewbtc, basewbtc.grv etc.) instead of their ibc equivalents.
# This feature isn't stable yet, and outputs will change in subsequent releases
```